src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.dtos
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@Schema(description = "Adjudication to migrate into service")
+data class AdjudicationMigrateDto(
+  @Schema(description = "The agency incident id")
+  val agencyIncidentId: Long,
+  @Schema(description = "The oic incident id")
+  val oicIncidentId: Long,
+  @Schema(description = "the offence sequence")
+  val offenceSequence: Long,
+  @Schema(description = "booking id on adjudication")
+  val bookingId: Long,
+  @Schema(description = "agency id on adjudication")
+  val agencyId: String,
+  @Schema(description = "the date and time of the incident")
+  val incidentDateTime: LocalDateTime,
+  @Schema(description = "the internal location id of the incident")
+  val locationId: Long,
+  @Schema(description = "the incident statement")
+  val statement: String,
+  @Schema(description = "adjudication under yoi rules")
+  val yoiRulesUsed: Boolean,
+  @Schema(description = "prisoner details")
+  val prisoner: MigratePrisoner,
+  @Schema(description = "offence details")
+  val offence: MigrateOffence,
+  @Schema(description = "optional victim", required = false)
+  val victim: MigrateVictim? = null,
+  @Schema(description = "incident associate", required = false)
+  val associate: MigrateAssociate? = null,
+)
+
+@Schema(description = "prisoner details")
+data class MigratePrisoner(
+  @Schema(description = "prisoner number on adjudication")
+  val prisonerNumber: String,
+  @Schema(description = "agency the prisoner is currently residing, or null if no longer in prison")
+  val currentAgencyId: String?,
+  @Schema(description = "gender of the prisoner")
+  val gender: String,
```

Comment:
would be good to know if this is mandatory in nomis, and what values it may contain

Reply:
It is mandatory. Do you need to persist it?  (it might change)
values from the reference table on dev
('SEX', 'F', null, null, 'Female', 2, 'Y', 'N', null);                                 
('SEX', 'M', null, null, 'Male', 1, 'Y', 'N', null);                                   
('SEX', 'NK', null, null, 'Not Known / Not Recorded', 3, 'Y', 'N', null);              
('SEX', 'NS', null, null, 'Not Specified (Indeterminate)', 4, 'Y', 'N', null);         
('SEX', 'REF', null, null, 'Refused', 5, 'N', 'N', TO_DATE('2006-07-12', 'YYYY-MM-DD'))

Reply:
thanks.  so we only have MALE / FEMALE but the reporting officer is prompted around "what they identify as" i think, will confirm with John what we default to for any non binary answers